Sprint-2/2-mandatory-debug/2.js

```diff
@@ -1,24 +1,31 @@
 // Predict and explain first...
-
-// Predict the output of the following code:
+// extracts only the last digit of a number.
+// Predict the output of the following code: 3
 // =============> Write your prediction here
 
-const num = 103;
+// let num = 103;
+
+// function getLastDigit(num) {
+//   return num.toString().slice(-1);
+// }
+
+// console.log(`The last digit of 42 is ${getLastDigit(42)}`);
+// console.log(`The last digit of 105 is ${getLastDigit(105)}`);
+// console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
-function getLastDigit() {
+// Now run the code and compare the output to your prediction
```

Comment:
You've corrected the mistake but your explanation isn't quite right. The problem was the number wasn't passed into the function as a parameter